core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -8634,7 +8491,7 @@ private void checkSemiJoinRuleOnAntiJoin(RelOptRule rule) {
 
   @Test void testExpandProjectInNullable() {
     final String sql = "with e2 as (\n"
-        + "  select empno, case when true then deptno else null end as deptno\n"
+        + "  select empno, case when deptno>0 then deptno else null end as deptno\n"
```

Comment:
```suggestion
        + "  select empno, case when deptno > 0 then deptno else null end as deptno\n"
```

Reply:
I've changed all occurrences of `deptno>0`.